config/hackathon.yml

```diff
@@ -16,6 +16,10 @@ defaults: &defaults
     Applicants must be 18 years of age by January 27th.
   thanks_for_applying: |
     Thanks for applying! You will be notified of your acceptance via email in January.
+  bus_captain_notes: |
+    * Only students on this list may board the bus. **No exceptions**- this is to ensure everyone goes through the application process & has signed RIT's waivers.
+    * If any emergencies arise, call our travel director, **Velizar Demirev**, at **(413) 221-2420**.
```

Comment:
Not yet correct for this year, but it won't show up until we start assigning bus captains in January so I think it's fine for now. Can update at a later date.